.travis.yml

```diff
@@ -3,6 +3,9 @@ cache: bundler
 before_install:
   - gem --version
   - gem list bundler
+  - sudo apt-get install python-matplotlib
+  - "export DISPLAY=:99.0"
+  - "sh -e /etc/init.d/xvfb start"
```

Comment:
why is this needed? :confused:

Reply:
I had run travis on my fork and had faced issue of not having any display parameter for running python script so I used this here for headless GUI for testing.
https://docs.travis-ci.com/user/gui-and-headless-browsers/

Reply:
Does that mean that you are running matplotlib in the normal unit tests? I would avoid that because the setup can be tricky and strongly depends on the environment. This doesn't give results well defined enough to be reliably checked in tests.

We have the integration tests which run matplotlib in a container: `integration/create_burndown_spec.rb` This works very reliably.

Reply:
I agree with @cornelius :+1:

Reply:
Yes if we have a docker container for matplotlib then it should be used,I will make the changes and also sorry for late response as I was busy with exams past week. :smile: